### ISSUE TITLE
Mage Talent Spell Power

### ISSUE THREAD

**Issue by ttghnw** [Atlantiss/NetherwingBugtracker#3269]
**Description**: 
Can you confirm that this is giving 50% crit damage increase for ALL spells.
On the same gear and around 950 frost dmg, frostbolts are critting for around 3.5k with and without the talent, this seemed present throughout the beta.

With the 100% from frost tree and 50% from arcane, there should be a noticeable difference.

The same with arcane - fireball builds, I don't see a big difference on the crits.

I believe it should be 

Please confirm this as I might be wrong.

**Comment by ircfelix** [comment]
Will check that later

**Comment by killablink** [comment]
![image](https://user-images.githubusercontent.com/43510874/49586532-6c446180-f92f-11e8-9510-a9a6df0c0be5.png)
With Spell Power talent 

![image](https://user-images.githubusercontent.com/43510874/49586546-77978d00-f92f-11e8-8452-1fbfdefdd5f6.png)
With no talents.

Seems to be working, just not sure how the crit damage and all applies

**Comment by rjorgensen** [comment]
Talent is working. I don't know how they should interact
I used rank 1 FB for speed to test it.
![untalented fb](https://user-images.githubusercontent.com/45169919/49582902-c0167100-f956-11e8-83a3-2634a24ff25d.png)
_fig 1. Untalented FB._ 
![spell power fb](https://user-images.githubusercontent.com/45169919/49582936-db817c00-f956-11e8-81fb-5d353783c4d4.png)
_fig 2. Spell Power FB._
![ice shards fb](https://user-images.githubusercontent.com/45169919/49582956-f05e0f80-f956-11e8-842d-50e3cc1be1cb.png)
_fig 3. Ice Shards FB._
![spellpower and ice shards fb](https://user-images.githubusercontent.com/45169919/49582974-feac2b80-f956-11e8-97bd-5f76bf62b612.png)
_fig 4. Ice Shards and Spell power FB._
![coefficients](https://user-images.githubusercontent.com/45169919/49587179-7718e980-f963-11e8-9fda-329500dafa66.png)
_table 1. Crit multipliers_
The talent is clearly working. However I am a little confused on how math works when you have both talents as the critical damage with both talents was lower compared to my calculations. 
(Maybe I need a brush up on percentage calculations)

**Comment by ircfelix** [comment]
TLDR: Bug report can be closed/is invalid.

Source:
http://web.archive.org/web/20080304073507/http://elitistjerks.com:80/f31/t18441-mage_sweet_informational_thread/p4/

> Only, for frost, the CSD pushes the crits from 200% to 209%, a 4,5% gain, due to funky mechanics (the formula: CSD crit multiplier = 1 + (1,03 * (base crit multiplier) -1) * (1 + talent modifier) = 1 + ( 1,03 * 1,5 -1) * 2 = 1 + 0,545 * 2 = 2,09.

> For Arc/Fire, we go from 245% to 254,45% crits, a 3,86% gain.
> For Arc/Frost, we go from 225% to 236,25%, a 5% gain.
> For Arcane, we go from 175% to 181,75%, a 3,86% gain.

This is some discussion about Chaotic Skyfire Diamond but it also contains the crit multipliers for the Arcane/Frost spec. Here we can see that an Arcane/Frost crit should do 2,25 times normal damage, Arcane/Fire 2,45 (includes ignite) and Arcane should do 1,75 times damage.

In our case the talent modifier from Frost only would be
> crit multiplier = 1 + (1 * (base crit multiplier) -1) * (1 + talent modifier)
> 1 + (1 * (1,5) -1) * (1 + 1) = 2

Did some testing with the 5/5 Ice Shards talent. The Frostbolts I was throwing were hitting from 1111-1146 damage and critting for around 2296 damage, which is exactly what the talent should give.
Taking the talent Spell Power into the boat, I get crits for ~2606, which is also exactly in line with what both talents should produce together.
